default.xml

```diff
@@ -31,5 +31,6 @@
   <project name="openembedded/meta-linaro" path="layers/meta-linaro" remote="linaro" revision="master"/>
   <project name="openembedded/meta-openembedded" path="layers/meta-openembedded" remote="github"/>
   <project name="openembedded/openembedded-core" path="layers/openembedded-core" remote="github"/>
+  <project name="meta-python2" path="layers/meta-python2" remote="oe"/>
```

Comment:
To match 'repo manifest -o', the entries must be sorted by names. "meta-python2" may not be below "openemb..."